docs/howto/operate/move-hub.md

```diff
@@ -106,6 +110,53 @@ do your modifications. This prevents needing to create another EC2
 instance just for this.
 ```
 
+### Azure Files
+
+```{note}
+The exception: even if the source and target cluster are in the same Azure subscription, re-using the same home directory storage might prove more difficult than copying files from one storage to the other.
+```
+
+We also use AzureFiles as in-cluster NFS storage. These are the steps to transfer the home directories between a NFS server located in another cluster, but in the same Azure subscription with an AzureFile NFS storage.
+
+AzureFile needs to be mounted in the source NFS VM in order to copy the data.
+
+1. Make sure AzureFile has access to the VNet in which the source NFS VM is in. Documentation on how to do that is can be found [here](https://docs.microsoft.com/en-us/azure/storage/files/storage-files-networking-endpoints?tabs=azure-portal#restrict-public-endpoint-access).
+
+2. On the source NFS VM:
+   1. Locate the private ssh key counterpart of the public one set for this cluster. The private key is usually encrypted with sops and stored into the repository where the infrastructure config of the cluster resides.
```

Comment:
General note that shouldn't block this PR.

We should update our Azure cluster creation docs to include a step that creates a new SSH key pair and uploads the sops-encrypted, private part to the `secrets` directory. Atm, I think Georgiana and I are using our own ssh keys which is unsustainable.

Reply:
> Atm, I think Georgiana and I are using our own ssh keys which is unsustainable.

Yessss! I guess we could still upload them to the `secrets` dir and generate new personal ones to fix this mhm... (But that kind of has consequences :grimacing: )

*I think* that if we change them now in terraform with a it will force a resource recreation (as per terraform's docs) of the cluster.

Absolutely  + 1 on documenting this!!!

Reply:
Tbh, right now there's more than just the SSH keys blocking other team members from working on the UToronto and CarbonPlan clusters 😅 so I wouldn't worry too much about those specific clusters now and instead focus on not generating future toil

Reply:
Done! Thank you! :sparkles: